lib/exception_notifier/views/exception_notifier/exception_notification.html.erb

```diff
@@ -45,7 +45,8 @@
         <tr>
           <td><%= raw summary %></td>
         </tr>
-      <% end %>
+        <% break if ObjectSpace.memsize_of(sections_content) > 4000000 && index == 9 %>
```

Comment:
Yes, you're right , we first wrote just an approximate value to 4mb but I'll fix it right no to be exactly 4mb :+1: